roles/openshift_health_checker/library/search_journalctl.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+"""Interface to journalctl"""
+
+from time import time
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+class InvalidMatcherRegexpException(Exception):
+    """Base exception class for invalid matcher regexp fields"""
+    pass
+
+
+class InvalidMatcherException(Exception):
+    """Base exception class for missing or invalid matcher fields."""
```

Comment:
None of these two need to be called "Base exception class", as they have no subclasses?

Suggestion: s/Base exception class/Exception